src/Model/CommonTypes.hs

```diff
@@ -68,15 +68,11 @@ computeScale :: Rectangle
 computeScale ((Position (lx, ly)), (Position (rx, ry)))
              (Dimensions (width, height)) = ((abs $ rx - lx) / width, (abs $ ry - ly) / height)
 
-data Hitbox = Hitbox
-  { hitboxPosition :: Position -- ^ Object position
-  , hitboxCollisionBoxes :: [Rectangle]
-  } deriving Show
-
 -- | Contains information about a movable game entity.
 data Object = Object
   { objectName :: String -- ^ Unique object identifier.
-  , objectHitbox :: Hitbox -- ^ Object hitbox.
+  , objectPosition :: Position -- ^ Object position
+  , objectCollisionBoxes :: [Rectangle]
```

Comment:
I like "objectHitbox" name more than "objectCollisionBoxes". It's shorter and clearer

Reply:
but "objectHitbox" less understandable. so I insist on leaving "objectCollisionBoxes"

Reply:
Ok